common/modules/gcp-project/audit.tf

```diff
@@ -0,0 +1,50 @@
+variable "nonce" {}
+
+variable "gcloud_timeout" {
+  default = 30
+}
+
+resource "null_resource" "add_audit_config" {
+  triggers = {
+    nonce = "${var.nonce}"
+  }
+
+  depends_on = [
+    "google_project_iam_policy.project",
+  ]
+
+  provisioner "local-exec" {
+    command = <<EOF
+      set -e
+
+      auditConfigs=$(echo '[${join(
+        ",",
+        formatlist(
+          "{\"auditLogConfigs\":[{\"logType\":\"DATA_READ\"},{\"logType\":\"DATA_WRITE\"}],\"service\":\"%s\"}",
+          concat(
+            var.audited_project_apis,
+            var.audited_apis
+          )
+        )
+      )}]' | jq -c -r -S .)
+
+      policy=$(timeout -t ${var.gcloud_timeout} gcloud projects get-iam-policy ${google_project.project.project_id} --format json)
+      policy_bindings=$(echo $policy | jq -c -r .bindings)
```

Comment:
How about `current_iam_policy_bindings`?

Reply:
It's a good practice to quote the shell variables to prevent issues with special characters.